### PR TITLE
Upgrade to GNOME 3.38 runtime

### PR DESCRIPTION
**Pull request by fedelibre**
Hey, I did not want to install the GNOME 3.34 runtime so I decided to build the flatpak and share it here.

I did a quick try with 3.38 also, but I got some errors when building gst-plugins-bad version 1.16.
I tried building version 1.18 (released few days ago), but the Sdk still has gstreamer 1.16 so I cannot compile it. I guess I should wait until gstreamer 1.18 enters the Sdk.

### CODE REVIEW

**Comment by fedelibre** [comment]
@bochecha Can you please review?

**Comment by hfiguiere** [comment]
maybe moving to 3.38 would be appropriate?

**Comment by fedelibre** [comment]
Ok, I will try again and push a 3.38 branch tonight

**Comment by bilelmoussaoui** [comment]
Please ping me once the update to 3.38 is done, I will merge it afterwards.

**Comment by fedelibre** [comment]
> I did a quick try with 3.38 also, but I got some errors when building gst-plugins-bad version 1.16.
> I tried building version 1.18 (released few days ago), but the Sdk still has gstreamer 1.16 so I cannot compile it. I guess I should wait until gstreamer 1.18 enters the Sdk.

I confirm what I wrote 20 days ago:

1. Building with 3.38 and gst-plugins-bad 1.16 produces some C errors. See attached log.
[PlayItSlowly-3.38-c-errors.txt](https://github.com/flathub/ch.x29a.playitslowly/files/5351373/PlayItSlowly-3.38-c-errors.txt)


2. Building with 3.38 and gst-plugins-bad 1.18 is not possible because freedesktop Sdk is still using gstreamer 1.16. I've pushed a [3.38 branch](https://github.com/fedelibre/ch.x29a.playitslowly/tree/3.38) in case someone wants to test it.

How to proceed?

**Comment by hfiguiere** [comment]
I think merging this now and then figuring out for 3.38 later would help.

@bilelmoussaoui ?

**Comment by hfiguiere** [comment]
Apparently glib-mkenums seems to generate invalid files...

**Comment by bilelmoussaoui** [comment]
> I think merging this now and then figuring out for 3.38 later would help.
> 
> @bilelmoussaoui ?

Sounds good to me. Especially that 3.34 is eol

**Comment by hfiguiere** [comment]
Building gst-plugins-bad with 3.38: 

```json
        {
            "name": "gstreamer-plugins-bad",
            "buildsystem": "meson",
            "config-opts": [
                "-Dbz2=disabled",
                "-Dcurl=disabled",
                "-Ddash=disabled",
                "-Ddecklink=disabled",
                "-Ddtls=disabled",
                "-Ddvb=disabled",
                "-Dexamples=disabled",
                "-Dfbdev=disabled",
                "-Dgl=disabled",
                "-Dhls=disabled",
                "-Dintrospection=disabled",
                "-Dkms=disabled",
                "-Dopenal=disabled",
                "-Drsvg=disabled",
                "-Dsmoothstreaming=disabled",
                "-Dsndfile=disabled",
                "-Dvulkan=disabled",
                "-Dwayland=disabled",
                "-Dwebp=disabled",
                "-Dorc=disabled",
                "-Dsoundtouch=enabled"
            ],
            "sources": [
                {
                    "type": "archive",
                    "url": "https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.16.2.tar.xz",
                    "sha256": "f1cb7aa2389569a5343661aae473f0a940a90b872001824bc47fa8072a041e74"
                }
            ],
            "cleanup": [
                "/include",
                "/lib/*.la",
                "/lib/gstreamer-1.0/*.la",
                "/lib/gstreamer-1.0/include",
                "/lib/pkgconfig",
                "/share/gtk-doc"
            ]
        },
```

tl;dr: use meson

**Comment by fedelibre** [comment]
> tl;dr: use meson

Oh, I didn't realise that 1.16 had also meson.build besides the autotools scripts.
Indeed it works using meson! I'm going to push the commit.

**Comment by fedelibre** [comment]
> Please ping me once the update to 3.38 is done, I will merge it afterwards.

@bilelmoussaoui Update to 3.38 is done

**Comment by bilelmoussaoui** [comment]
Thanks!
